sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -553,3 +553,57 @@ def create(factory, transform_id, transform_proto, unused_parameter, consumers):
           factory.state_sampler),
       transform_proto.unique_name,
       consumers)
+
+
+@BeamTransformFactory.register_urn(
+    urns.PRECOMBINE_TRANSFORM, beam_runner_api_pb2.CombinePayload)
+def create(factory, transform_id, transform_proto, payload, consumers):
+  # TODO: Combine side inputs.
```

Comment:
<!--new_thread; commit:15c1178c61218f4f56ad3d74a1d52f3f1dc6e727; resolved:0-->
Do we want to link a JIRA with a description of a TODO?

Reply:
We're actually talking about removing this in Java, so I don't think it's worth surfacing there for now. The SDK API doesn't even support this for now, this is mostly a marker for if this ever gets added. Changed to a normal comment.